package com.kpiroom.bubble.util.exceptions.core.db

class DbCancelledException : DbException("Call cancelled")